src/test/resources/soop_sst_trajectory.xml

```diff
@@ -6,68 +6,849 @@
+    </attribute>
+    <attribute name="data_centre" value="'eMarine Information Infrastructure (eMII)'"/>
+    <attribute name="data_centre_email" value="'info@emii.org.au'"/>
+    <attribute name="institution_references" value="'http://www.imos.org.au/emii.html'"/>
 
+    <attribute name="citation">
+      <value>'The citation in a list of references is: "IMOS [year-of-data-download], [Title], [data-access-url], accessed [date-of-access]"'</value>
+    </attribute>
+    <attribute name="acknowledgement">
+      <value>'Any users of IMOS data are required to clearly acknowledge the source of the material in the format: "Data was sourced from the Integrated Marine Observing System (IMOS) - an initiative of the Australian Government being conducted as part of the National Collaborative Research Infrastructure Strategy and and the Super Science Initiative."'</value>
+    </attribute>
+    <attribute name="distribution_statement">
+      <value>'Data may be re-used, provided that related metadata explaining the data has been reviewed by the user, and the data is appropriately acknowledged. Data, products and services from IMOS are provided "as is" without any warranty as to fitness for a particular purpose.'</value>
+    </attribute>
+    <attribute name="project_acknowledgement">
+      <value>'The collection of this data was funded by IMOS'</value>
+    </attribute>
+  </globalattributes>
+  <variables>
     <variable>
-      <name>id</name>
+      <name>trajectory</name>
       <encoder>integer</encoder>
       <attributes>
-        <attribute name="_FillValue" value="99999"/>
+        <attribute name="long_name" value="Unique identifier for each feature instance"/>
+        <attribute name="cf_role" value="trajectory_id"/>
       </attributes>
     </variable>
-
     <variable>
       <name>TIME</name>
-      <encoder>time</encoder>
+      <encoder>time</encoder>  
       <dimensions>
+      <!--  SO i am  following what's in the anmn_ts template but i  questioning the encoder thta is used here 'time'. NODC recommends time to be of type double, which is also the current IMOS convention. Why using type 'time' then (except the fact that TIME  is a timestamp in the DB). TIME should be converted back to double to conform the NODC format? Am i missing something here?
+      -->
         <dimension name="TIME"/>
       </dimensions>
       <attributes>
-        <attribute name="_FillValue" value="99999."/>
+        <attribute name="standard_name" value="'time'"/>
+        <attribute name="long_name" value="'time'"/>
         <attribute name="units" value="'days since 1950-01-01 00:00:00 UTC'"/>
+        <attribute name="calendar" value="'gregorian'"/>
+        <attribute name="axis" value="'T'"/>
+        <attribute name="_FillValue" value="-999999."/>   
+        <attribute name="valid_min" value="0."/>
+        <attribute name="valid_max" value="90000."/>
+        <attribute name="ancillary_variables" value="'TIME_quality_control'"/>
```

Comment:
I personally don't use QC info for coordinate variables. If the spatio-temporal location of the data is not good I don't see the point of storing the data.